python/pyiceberg/catalog/hive.py

```diff
@@ -229,24 +290,43 @@ def create_table(
             ValueError: If the identifier is invalid
         """
         database_name, table_name = self.identifier_to_database_and_table(identifier)
-        current_time_millis = int(time.time())
+        seconds_since_epoch = int(time.time())
+
+        location = self._resolve_table_location(location, database_name, table_name)
+
+        metadata_location = f"{location}metadata/{uuid.uuid4()}.metadata.json"
+        metadata = TableMetadataV2(
+            location=location,
+            schemas=[schema],
+            current_schema_id=schema.schema_id,
+            partition_specs=[partition_spec],
+            default_spec_id=0,
+            sort_orders=[sort_order],
+            default_sort_order_id=0,
```

Comment:
Sort order ID 0 is reserved for the unsorted order. This can't always assume that the order ID is 0. Like the above, I think this should be `sort_order.order_id`.

Reply:
Great catch, thanks!